templates/ecommerce/src/payload/collections/Pages/index.ts

```diff
@@ -17,7 +17,7 @@ export const Pages: CollectionConfig = {
     useAsTitle: 'title',
     defaultColumns: ['title', 'slug', 'updatedAt'],
     preview: doc => {
-      return `${process.env.PAYLOAD_PUBLIC_SERVER_URL}/api/preview?url=${encodeURIComponent(
+      return `${process.env.PAYLOAD_PUBLIC_SERVER_URL}/next-api/preview?url=${encodeURIComponent(
```

Comment:
What do you think about renaming this to just `next`? Just a thought.

Reply:
changed it to next since it's more succinct